cmd/integration_tests.go

```diff
@@ -114,7 +114,6 @@ func (c *IntegrationTestsCommand) Command() *cobra.Command {
 			CustomSetValue: cmdUtils.SetRegistrationContactType,
 			ConfigKey:      &integrationTestsOpts.RegistrationContactType,
 			Required:       true,
-			FlagDefault:    data.RegistrationContactTypePhone.String(),
```

Comment:
I removed this default value to enforce users to explicitly pass the value they want.